src/index.js

```diff
@@ -53,10 +56,12 @@ const displayTasks = () => {
     editTaskInput.className = 'invisible';
     editTaskInput.type = 'text';
     editTaskInput.value = task.description;
-    editTaskInput.addEventListener('focusout', (e) => {
-      toDoItem.classList.toggle('set-focus-bg');
-      updateTask(tasks, index, e.target.value);
-      displayTasks();
+    editTaskInput.addEventListener('keypress', (e) => {
```

Comment:
- [ ] When I refresh the page, the check box of the completed tasks is not checked , i think a misspelling of `completed` made this issue, kindly fix it by correcting `completeted` to `completed` : 



bug preview: 

![buggcheck](https://user-images.githubusercontent.com/89282221/221369437-ee2c59e5-1100-473f-b71a-c1d677e4ce05.gif)


current code : 
```JS
 if (task.completeted) {
      taskInput.setAttribute('checked', 'true');
    }
```

quick fix  : 
```JS
 if (task.completed) {
      taskInput.setAttribute('checked', 'true');
    }
```